src/package.json

```diff
@@ -112,13 +113,13 @@
     "react-router-redux": "^4.0.8",
     "react-table": "^7.7.0",
     "react-transition-group": "^2.9.0",
-    "redux": "^3.7.2",
+    "redux": "^4.2.0",
     "redux-auth-wrapper": "^2.0.3",
-    "redux-devtools-extension": "^2.13.7",
-    "redux-thunk": "^2.3.0",
+    "redux-thunk": "^2.4.1",
     "reselect": "^4.1.4",
     "sass": "^1.34.1",
     "ssri": "^6.0.2",
+    "url": "^0.11.0",
```

Comment:
Why does the `url` package need to be introduced as part of the upgrade? It seems to very popular but at the same time hasn't been updated for seven years, and Node has surely moved on in that time with its own implementation.